pkg/controller/certificates/approver/sarapprove_test.go

```diff
@@ -29,12 +29,18 @@ import (
 
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	testclient "k8s.io/client-go/testing"
 	authorization "k8s.io/kubernetes/pkg/apis/authorization/v1beta1"
 	capi "k8s.io/kubernetes/pkg/apis/certificates/v1beta1"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset/fake"
+	"k8s.io/kubernetes/pkg/features"
 )
 
+func init() {
+	utilfeature.DefaultFeatureGate.Set(string(features.RotateKubeletServerCertificate) + "=true")
```

Comment:
I wasn't expecting this... the approver is permission based... I only expected the kubelet rotation to be gated by the flag. Also, setting this one way in an init() function means it is always enabled when testing, right?